apps/web/data/docs-sidebar.ts

```diff
@@ -67,6 +67,7 @@ export const DOCS_SIDEBAR: DocsSidebarSection[] = [
       { title: "Dropdown", href: "/docs/components/dropdown" },
       { title: "Footer", href: "/docs/components/footer" },
       { title: "Forms", href: "/docs/components/forms" },
+      { title: "HR", href: "/docs/components/hr", isNew: true },
```

Comment:
This should be actually under `typography` section

Reply:
Gotcha!.. I wasn't aware of it, so added it here... Will Change it,